package.json

```diff
@@ -73,6 +72,6 @@
   ],
   "contributors": [
     "Eric Liu (https://github.com/metonym)",
-    "Josef Aidt (https://github.com/josefaidt)"
```

Comment:
I'd prefer to preserve existing users on this list. `josefaidt` was a maintainer at IBM.